unreleased_history/behavior_changes/universal_level_assgiment.md

```diff
@@ -0,0 +1 @@
+*Assigning levels for external files are done in the same way for universal compaction and leveled compaction.
```

Comment:
I think it's useful to explain that the old behavior assigns files to L0 and the new behavior assigns files to the lowest level possible, which helps to improve read performance.